qt/applications/workbench/workbench/app/start.py

```diff
@@ -86,6 +86,15 @@ def initialize():
         # https://www.tornadoweb.org/en/stable/#installation
         import asyncio
         asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
+    if sys.platform == 'darwin':
+        # Force layer-backing on macOS >= Big Sur (10.16)
+        # or the application hangs
+        # A fix inside Qt is yet to be released:
+        # https://codereview.qt-project.org/gitweb?p=qt/qtbase.git;a=commitdiff;h=c5d904639dbd690a36306e2b455610029704d821
+        from distutils.version import LooseVersion
+        import platform
+        if LooseVersion(platform.mac_ver()[0]) >= '11':
+            os.environ['QT_MAC_WANTS_LAYER'] = '1'
```

Comment:
Although I'm on version 11, I do not enter this if statement on my computer.

Reply:
Thanks for finding this. Time for a workaournd on top of the workaround :(

I think we'll check for both 11 & 10.16 and that should cover it.